classes/assets/StylesheetManager.php

```diff
@@ -53,11 +53,16 @@ public function register(
         $media = self::DEFAULT_MEDIA,
         $priority = self::DEFAULT_PRIORITY,
         $inline = false,
-        $server = 'local'
+        $server = 'local', 
+        $needRtl = true
     ) {
         if ('remote' === $server) {
             $this->add($id, $relativePath, $media, $priority, $inline, $server);
-        } elseif ($fullPath = $this->getFullPath($relativePath)) {
+        }
+        //RTL : search and use rtl css file if it's not originally rtl
+        else if ($needRtl && Context::getContext()->language->is_rtl && $fullPath = $this->getFullPath(preg_replace('/(^[^.].*)(\.css)$/', '$1_rtl.css', $relativePath))) {
```

Comment:
This line breaks the tests (https://travis-ci.org/PrestaShop/PrestaShop/jobs/261803713#L875)

Also, an affectation in a `if(...)` is a bad idea, we could think you forgot an equal sign. Please try to move it outside.

Finally, wouldn't a `str_replace('.css', '_rtl.css', $relativePath)` be enough? This function costs lest CPU resources than a `preg_replace`.